.travis.yml

```diff
@@ -85,16 +85,12 @@ jobs:
         - sudo mkdir -p /usr/share/codespell && sudo ln -s
            /usr/lib/python3/dist-packages/codespell_lib/data/dictionary.txt
            /usr/share/codespell/dictionary.txt
-        # TODO: stop screen-scraping the git log and switch to
-        # ${TRAVIS_COMMIT_RANGE} or ${TRAVIS_BRANCH}..HEAD after testing
-        # both force pushes and not.
+
         # Note TRAVIS_COMMIT_RANGE has triple dots for "git diff" and
         # needs conversion to double dots for git log's inconsistent
         # user interface.
-        - printf '%s %s\n' "${TRAVIS_BRANCH}" "${TRAVIS_COMMIT_RANGE/.../..}"
-        - SHA_PR=`git log --oneline -1 | sed -rn "s/.*Merge (.*) into.*/\1/p"`
-        - SHA_MAIN=`git log --oneline -1 | sed -rn "s/.*Merge .* into (.*)/\1/p"`
-        - ( set -x; scripts/checkpatch.pl --strict --codespell -g $SHA_MAIN..$SHA_PR )
+        - ( set -x; scripts/checkpatch.pl --strict --codespell
+                     -g "${TRAVIS_COMMIT_RANGE/.../..}" )
```

Comment:
OK. so here we still use `origin/master..pull_request` for checkpatch instead of `origin/master...pull_request`.
This would be good trick to just replace three dot to two dot. Can you add this into the commit message. I do not think we use `TRAVIS_COMMIT_RANGE` directly here as in you comments.

Reply:
Done, commit message updated. No other change.